src/h3lib/include/h3api.h.in

```diff
@@ -308,15 +308,15 @@ void H3_EXPORT(getPentagonIndexes)(int res, H3Index *out);
  * Functions for h3GetResolution
  * @{
  */
-/** @brief returns the resolution of the provided hexagon */
+/** @brief returns the resolution of the provided H3 cell */
 int H3_EXPORT(h3GetResolution)(H3Index h);
 /** @} */
 
 /** @defgroup h3GetBaseCell h3GetBaseCell
  * Functions for h3GetBaseCell
  * @{
  */
-/** @brief returns the base cell of the provided hexagon */
+/** @brief returns the base cell "number" (0 to 121) of the provided H3 cell */
 int H3_EXPORT(h3GetBaseCell)(H3Index h);
 /** @} */
```

Comment:
Should be valid for both cells and edges?

Reply:
Kinda. It will return the base cell of the origin hexagon, but there's an edge case (pun not intended, but now kinda is) where the destination is on a different base cell. If we bring up edges we should note that it is origin only.

Reply:
Good point. I'll leave it as `cell` for now.